lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorType.java

```diff
@@ -30,6 +30,9 @@
    * Indicates an Evaluator that runs on the CLR.
    */
   CLR,
+  /** Indicates an Evaluator that runs on a local thread.
+   */
+   LOCAL,
```

Comment:
Yes it only applies to local runtime and is a configuration option by which client, driver and evaluators are all run as thread inside the same process (JVM for the moment). This is more like  an option for the actual EvaluatorType and is currently used in `AllocatatedEvaluatorImpl#makeEvaluatorConfiguration` to generate the proper configuration.